target/linux/ath79/dts/ar9344_comfast_cf-e120a-v3.dts

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	compatible = "comfast,cf-e120a-v3", "qca,ar9344";
+	model = "COMFAST CF-E120A v3";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &wan;
+		led-failsafe = &wan;
+		led-upgrade = &wan;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&led_rssimediumhigh_pin>;
+
+		wan: wan {
+			label = "cf-e120a-v3:green:wan";
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+		};
+
+		lan {
+			label = "cf-e120a-v3:green:lan";
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+		};
+
+		rssilow {
+			label = "cf-e120a-v3:red:rssilow";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+
+		rssimediumlow {
+			label = "cf-e120a-v3:red:rssimediumlow";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		rssimediumhigh {
+			label = "cf-e120a-v3:green:rssimediumhigh";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+		};
+
+		rssihigh {
+			label = "cf-e120a-v3:green:rssihigh";
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	ath9k-leds {
+		compatible = "gpio-leds";
+
+		wlan {
+			label = "cf-e120a-v3:green:wlan";
+			gpios = <&gpio 0 GPIO_ACTIVE_LOW>;
```

Comment:
Hm, the separate "ath9k-leds" node should be used, if the LED specified there is really connected
through an external ath9k-chip's gpio.

the gpio-phandle you specified:

     gpios = <&gpio 0 GPIO_ACTIVE_LOW>;

makes it look like the LED is controlled via GPIO0 of the SoC's main pinctrl.

(If it's on gpio0 of the SoC, you can forgo this extra node and add it above to
the other LEDs in the "leds" node.)

Reply:
Thanks for the review, @chunkeey

Yes, the wlan LED is solely controlled by the SoC's GPIO0. I've therefore moved it to the leds node as you suggested.